app/index.html

```diff
@@ -60,27 +60,27 @@
       </div>
     </div>
 
-    <div ng-controller='FullscreenController as fcCtrl' ng-if='::$root.fullscreen || undefined' ng-show='$root.fullscreen && $root.tab === 1'>
```

Comment:
I always read that `fcCtrl` as fuckController, please don't change this, my life will be ruined! :smile: